Source/Core/MortonOrder.js

```diff
@@ -0,0 +1,85 @@
+import Check from "./Check.js";
+import defined from "./defined.js";
+import DeveloperError from "./DeveloperError.js";
+
+/**
+ * Morton Order (aka Z-Order Curve) helper functions.
+ * @see {@link https://en.wikipedia.org/wiki/Z-order_curve}
+ * @see {@link http://graphics.stanford.edu/~seander/bithacks.html#InterleaveBMN}
```

Comment:
Could we move this reference about bit interleaving into the `removeSpacingBetweenBits`? feel like it's more helpful closer to the implementation